game/characters.py

```diff
@@ -1,17 +1,31 @@
+class Character:
+    Character = []
 
-class Player:
+    def __init__(self, hp: int, damage: int, defence: int, room=None):
+        self.hp = hp
+        self.damage = damage
+        self.defence = defence
+        self.room = room
+
+
+class Player(Character):
+    Player = []
 
-    def __init__(self, name: str, hp=100, damage=1, defence=1):
+    def __init__(self, name: str, hp: int, damage: int, defence: int, room=None):
+        super().__init__(hp, damage, defence, room)
         self.name = name
         self.hp = hp
         self.damage = damage
         self.defence = defence
```

Comment:
Kdyz delame super().__init__ ... tak tim vlastne prehazujeme tu praci na Parent Class - to je spravnej pristup... ale potom uz neni nutne nastavovat ty atributy znovu - ty atributy se nastavi v parent classe, jedinej atribut ktery by se tu mel nastavovat je 'name'

Reply:
@PetrLegen88

Reply:
ok tohle jsem už upravil

Reply:
super!